deployments/multiple-ragapps/docker-compose.yml

```diff
@@ -31,9 +79,9 @@ services:
       - traefik.http.services.ragapp-cs50.loadbalancer.server.port=8000
```

Comment:
how about we remove these cs50 and cs100 examples - the user can create their own with the manager

Reply:
Of course, cs50 and cs100 are just examples and should not include here. but i just keep them here first after our configuration is stable then we can easily add it to the manager image and remove them